src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-discovery-modal/iscsi-target-discovery-modal.component.html

```diff
@@ -26,7 +26,7 @@
             <span class="invalid-feedback"
                   *ngIf="discoveryForm.showError('user', formDir, 'pattern')"
                   i18n>User names must have a length of 8 to 64 characters and can contain
-              alphanumeric characters, '.', '@', '-', '_' or ':'.</span>
+              alphanumeric characters, '.', '&#64;', '-', '_' or ':'.</span>
```

Comment:
This is not very dev-friendly, but it looks like it's not easy:
```suggestion
              alphanumeric characters, '.', '{{'@'}}', '-', '_' or ':'.</span>
```